executor/runtime/types/types_test.go

```diff
@@ -23,10 +27,71 @@ func TestEntrypointParse(t *testing.T) {
 		t.Fatal("Can't parse expected result JSON", err)
 	}
 
-	result, err := c.GetEntrypointFromProto()
+	result, cmd, err := c.Process()
 	if err != nil {
 		t.Fatalf("Can't parse entrypoint %q: %v", *c.TitusInfo.EntrypointStr, err)
 	}
 	assert.EqualValues(t, result, expectedSlice)
+	assert.Nil(t, cmd)
+}
+
+func TestCustomCmd(t *testing.T) {
+	t.Run("WithNilEntrypoint", testCustomCmdWithEntrypoint(nil))
+	t.Run("WithEmptyEntrypoint", testCustomCmdWithEntrypoint([]string{}))
+	t.Run("WithEntrypoint", testCustomCmdWithEntrypoint([]string{"/bin/sh", "-c"}))
+}
 
+func testCustomCmdWithEntrypoint(entrypoint []string) func(*testing.T) {
+	return func(t *testing.T) {
+		c := Container{
+			TitusInfo: &titus.ContainerInfo{
+				Process: &titus.ContainerInfo_Process{
+					Entrypoint: entrypoint,
+					Command:    []string{"sleep", "1"},
+				},
+			},
+		}
+
+		entrypoint, cmd, err := c.Process()
+		if err != nil {
```

Comment:
`require.NoError(err)`?

Reply:
no other tests here are using that lib. I'd personally rather not to, unless you feel strongly about it.